src/utils/makeReactNativeConfig.js

```diff
@@ -372,22 +372,28 @@ function makeReactNativeConfig(
   return webpackConfig;
 }
 
-function injectPolyfillIntoEntry({
-  entry: userEntry,
-  root,
-  initializeCoreLocation = 'node_modules/react-native/Libraries/Core/InitializeCore.js',
-}: {
-  entry: WebpackEntry,
-  root: string,
-  initializeCoreLocation?: string,
-}) {
+function injectPolyfillIntoEntry(
+  {
+    entry: userEntry,
+    root,
+    initializeCoreLocation = 'node_modules/react-native/Libraries/Core/InitializeCore.js',
+  }: {
+    entry: WebpackEntry,
+    root: string,
+    initializeCoreLocation?: string,
+  },
+  withHotPatch: boolean = true
```

Comment:
Instead of passing second argument, I'd pass the dev variable to the config object  and use that to determine whether add hot patch or not

Reply:
Oh, got it. Will make that change...
I assume you mean like this:
```
entry: injectPolyfillIntoEntry(
        {
          root: options.root,
          initializeCoreLocation: options.initializeCoreLocation,
          entry,
	  dev: options.dev,
        }
      ),
```